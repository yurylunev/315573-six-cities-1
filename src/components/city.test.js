import React from "react";
import renderer from "react-test-renderer";
import City from "./city.jsx";

it(`CitiesList correctly renders`, () => {
  const tree = renderer
    .create(<City city={`City`} clickHandler={jest.fn()} activeClassName={`activeClassName`} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
