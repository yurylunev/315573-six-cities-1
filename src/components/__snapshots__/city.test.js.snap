// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList correctly renders 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
  >
    <span />
  </a>
</li>
`;
